test/start.test.js

```diff
@@ -556,7 +556,7 @@ test('should start the server listening on 0.0.0.0 when running in kubernetes',
   t.pass('server closed')
 })
 
-test('should start the server with watch options that the child process restart when directory changed', { skip: process.platform === 'win32' }, async (t) => {
+test('should start the server with watch options that the child process restart when directory changed', { skip: process.platform === 'win32' || (process.platform === 'darwin' && ['v20', 'v19', 'v18'].some(v => process.version.startsWith(v))) }, async (t) => {
```

Comment:
We skip this on windows. Can we skip on macos?

Reply:
go ahead

Reply:
I would prefer not to tbh, but I don't have good arguments for it.

Reply:
I'll spend some time to try and debug. I don't understand what causes failure here tbh